tests/bot/cogs/test_logging.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from unittest.mock import patch
+
+from bot import constants
+from bot.cogs.logging import Logging
+from tests.helpers import MockBot, MockTextChannel
+
+
+class LoggingTests(unittest.IsolatedAsyncioTestCase):
+    """Test cases for connected login."""
+
+    def setUp(self):
+        self.bot = MockBot()
+        self.cog = Logging(self.bot)
+        self.dev_log = MockTextChannel(id=1234, name="dev-log")
+
+    @patch("bot.cogs.logging.DEBUG_MODE", False)
+    async def test_debug_mode_false(self):
+        """Should send connected message to dev-log."""
+        self.bot.get_channel.return_value = self.dev_log
+
+        await self.cog.startup_greeting()
+        self.bot.wait_until_guild_available.assert_awaited_once_with()
+        self.bot.get_channel.assert_called_once_with(constants.Channels.dev_log)
+
+        embed = self.dev_log.send.call_args[1]['embed']
+        self.dev_log.send.assert_awaited_once_with(embed=embed)
```

Comment:
If you retrieved the embed from the call args, then you already know it's been called with that value. It doesn't make sense to use a value from its call args in such an assertion. In such case, I would advice you just use `assert_awaited_once()`.